components/director/internal/open_resource_discovery/fixtures_test.go

```diff
@@ -725,7 +736,9 @@ func fixBundles() []*model.Bundle {
 			ApplicationID:                appID,
 			Name:                         "BUNDLE TITLE",
 			Description:                  str.Ptr("lorem ipsum dolor nsq sme"),
+			Version:                      str.Ptr("1.1.2"),
```

Comment:
Same.
It definitely can be used in tests, but is strange that we anyway allow invalid values to be persisted when API is used and perform  input validation only when we are scrapping the ORD data. 
Not something that should be fixed in this PR, but something that can be improved in the future.